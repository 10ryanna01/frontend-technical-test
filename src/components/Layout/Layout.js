export default function Layout({ children }) {
  return (
    <>
      <main data-testid="layoutTest1" className="UI__main-layout">
        {/* build out default app layout with this component */}

        {children}
      </main>
    </>
  );
}
