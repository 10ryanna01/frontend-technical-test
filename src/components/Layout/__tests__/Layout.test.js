import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Layout from '../Layout';

it('should render an initial Layout component', () => {
  render(<Layout />);

  const layoutTest1 = screen.getByTestId('layoutTest1');
  expect(layoutTest1).toBeInTheDocument();
});
it('takes a snapshot of Layout component', async () => {
  const tree = renderer.create(<Layout />);
  expect(tree).toMatchSnapshot();
});
