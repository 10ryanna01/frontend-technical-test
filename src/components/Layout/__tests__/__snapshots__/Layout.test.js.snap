// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Vehicle correctly 1`] = `
<div
  data-testid="layoutTest1"
/>
`;

exports[`takes a snapshot of Layout component 1`] = `
<div
  data-testid="layoutTest1"
/>
`;
